resources/views/livewire/price-ticker.blade.php

```diff
@@ -1,10 +1,10 @@
 @php ($isDisabled = ! Network::canBeExchanged() || config('explorer.network') !== 'production' || ! $isAvailable)
 
 <div
-    wire:poll.60s
+    wire:poll.visible.60s
     class="w-full md:w-auto"
     :class="{ 'opacity-50': busy }"
-    x-data="{ to: '{{ $to }}', busy: false }"
```

Comment:
why was this change needed

Reply:
`to` isn't referenced anywhere with alpine anymore. it wasn't prior to my change that I can see either